vendor/TraceKit/tracekit.js

```diff
@@ -35,6 +34,13 @@ TraceKit.wrap = function traceKitWrapper(func) {
     return wrapped;
 };
 
+TraceKit.getLocationHref = function () {
+    if (typeof document === 'undefined')
+        return '';
+
+    return document.location.href;
```

Comment:
Possibly ... but it wouldn't surprise me if `location` is similarly undefined. I'll check.